web-common/src/components/virtualized-table/sections/PinnedColumns.svelte

```diff
@@ -1,10 +1,10 @@
 <script lang="ts">
-  import type { VirtualizedTableColumns } from "../../../types";
   import { createEventDispatcher } from "svelte";
   import Cell from "../core/Cell.svelte";
   import ColumnHeader from "../core/ColumnHeader.svelte";
   import Row from "../core/Row.svelte";
   import type { PinnedColumnSide } from "../types";
+  import type { VirtualizedTableColumns } from "@rilldata/web-local/lib/types";
```

Comment:
Can we move this type into `web-common`? `web-common` shouldn't be importing from `web-local`